test/src/integration/integration_test.go

```diff
@@ -70,6 +71,8 @@ func TestEndToEndK8s(t *testing.T) {
 	os.Setenv("TEST_TYPE", "k8s")
 
 	defer test_structure.RunTestStage(t, "teardown", func() {
+		runHelmDelete(t)
```

Comment:
Does it just go to the next step even if it fails to execute `helm delete` due to, for example, lack of the command?

Reply:
Yes, I tested it when ssh connection was not possible.
```
TestEndToEndK8s 2021-01-25T11:46:59+09:00 ssh.go:416: Running command helm delete prod on centos@54.211.213.216                                                                               TestEndToEndK8s 2021-01-25T11:47:09+09:00 integration_test.go:86: Destroying <kubernetes> Infrastructure
TestEndToEndK8s 2021-01-25T11:47:09+09:00 retry.go:72: terraform [destroy -auto-approve -input=false -lock=false -no-color]                                                                   TestEndToEndK8s 2021-01-25T11:47:09+09:00 command.go:87: Running command terraform with args [destroy -auto-approve -input=false -lock=false -no-color]
```

Reply:
👍  Thank you!